### ISSUE TITLE
Contrast Minimum Accessibility

### ISSUE THREAD

**Issue by irmerk** [accordproject/cicero-ui#57]
# WCAG 2.0 Level AA [1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) (Contrast (Minimum))

## Tasks
- Contrast ratio should be at least `4.5:1`.

## Related Issues: 
- Issue [#42](https://github.com/accordproject/template-studio-v2/issues/42) in Template Studio v2

**Comment by irmerk** [comment]
Scan through code and modules to find anything not complying with this guideline

Issues found should be brought to the attention of @Michael-Grover

**Comment by irmerk** [comment]
### Hacktoberfest:
This issue can possibly provide for multiple pull requests.

**Comment by sameshl** [comment]
@irmerk You can assign me this issue. I would love to work on this. Could you just elaborate a bit on how exactly this needs to be done?

Thanks!

**Comment by irmerk** [comment]
Sure @sameshl, I think I would be looking into guides online on how to be compliant with WCAG 2.1 Level AA 1.4.3.

**Comment by irmerk** [comment]
Opening this back up after 14 days stale.

**Comment by starblind** [comment]
![image](https://user-images.githubusercontent.com/453473/67629759-319e2d00-f851-11e9-91b3-9e642c3e44b1.png)

Add to Contract - 2.5

**Comment by starblind** [comment]
![image](https://user-images.githubusercontent.com/453473/67629784-a6716700-f851-11e9-91e6-e89ecf7fd795.png)

The background color on hover in the style menu - 1.19-1.5

**Comment by starblind** [comment]
![image](https://user-images.githubusercontent.com/453473/67629787-df114080-f851-11e9-99a1-69fefe287ba6.png)

Clause title block - 2.68 - should be at least 3

**Comment by CDharmateja** [comment]
@starblind Thanks for mentioning. There are few more contrast issues. I was actually working on this issue in Template Studio v2 repo. Most of them are to be done in this repo. Are you working on this?

**Comment by starblind** [comment]
nope, I just reported it

**Comment by elit-altum** [comment]
Hey @irmerk I'd like to work on this. Could you please assign it to me ?
